flash/core/utilities/imports.py

```diff
@@ -60,6 +60,11 @@ def _compare_version(package: str, op, version) -> bool:
         return False
     try:
         pkg_version = Version(pkg.__version__)
+    except AttributeError:
+        # In case the module doesn't have __version__ attribute (example: baal)
+        import pkg_resources
+
+        pkg_version = Version(pkg_resources.get_distribution("baal").version)
```

Comment:
Let's update this to use the more flexible version from here (just copy the code across I guess): https://github.com/PyTorchLightning/pytorch-lightning/blob/1a502c061cf43f60cbf0f2089981ef6b8a57419c/pytorch_lightning/utilities/imports.py#L62

That way we don't hardcode the `baal` in there

Reply:
My bad, should not have hardcoded it there.

Reply:
Should be fixed now!